### PR TITLE
Feature/dashboard

### PR DESCRIPTION
**Pull request by sheldonkwoodward**

### CODE REVIEW

File: src/app/shared/pages-scroll-cards/pages-scroll-cards.component.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<div class="card-columns row">
+  <div *ngFor="let page of requestData" class="card p-4 text-center col-3">
+    <!-- content -->
+    <div class="card-block">
+      <h4 class="card-title"><a routerLink="{{page.url}}">{{page['title']}}</a></h4>
+      <p class="card-text">{{page['description']}}</p>
+    </div>
+    <!-- metadata -->
+    <!-- TODO: not working -->
+    <div *ngIf="showMeta == true" class="card-block">
```
Comment (sheldonkwoodward) [inline src/app/shared/pages-scroll-cards/pages-scroll-cards.component.html]: @asteroidice this is the line that doesn't work. showMeta is a variable in pages-scroll-cards.component.ts.

Comment (asteroidice) [reply src/app/shared/pages-scroll-cards/pages-scroll-cards.component.html]: Yeah, that's a fun one. I'm not completely sure why it's not working.

File: src/app/routes/dashboard/dashboard.component.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<h1>Dashboard</h1>
+
+<h2>Search</h2>
+<input />
+<button>Search</button>
+
+<h2>Featured</h2>
```
Comment (asteroidice) [inline src/app/routes/dashboard/dashboard.component.html]: Hide this if there aren't any featured articles.

**Comment by asteroidice** [comment]
Also, could you make events show up under featured? I think that would be more intuitive.

File: src/app/shared/pages-scroll-cards/pages-scroll-cards.component.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  selector: 'pages-scroll-cards',
+  templateUrl: './pages-scroll-cards.component.html',
+  styleUrls: ['./pages-scroll-cards.component.css']
+})
+
+export class PagesScrollCardsComponent {
+  @Input() requestData: any;
+  @Input() showMeta: boolean;
+
+  getAuthor(page) {
```
Comment (asteroidice) [inline src/app/shared/pages-scroll-cards/pages-scroll-cards.component.ts]: This is a great option for now and for the initial release of this project. We could look into the possibility of fetching a thumbnail of the person's profile photo later.

Comment (sheldonkwoodward) [reply src/app/shared/pages-scroll-cards/pages-scroll-cards.component.ts]: Awesome!

File: src/app/routes/dashboard/dashboard.component.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+})
+export class DashboardComponent {
+  featureds: any = [];
+  events: any = [];
+  categories: any = [];
+  departments: any = [];
+
+  constructor(private requestService: RequestService, private route: ActivatedRoute, private router: Router) {
+    // get featureds data
+    this.requestService.get('/pages/featureds', (data) => {
```
Comment (asteroidice) [inline src/app/routes/dashboard/dashboard.component.ts]: Later we might look into generalizing these. But for now, there are many, more important things we need to accomplish.

**Pull request merged**